### ISSUE TITLE
Header images lack alt text

### ISSUE THREAD

**Issue by jenweber** [empress/empress-blog-casper-template#19]
How can we add alt text to the image headers? I wasn't able to see how image information gets passed/rendered, but I can help with this if you point me in the right direction.
